module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -621,7 +621,8 @@ public function getMyProfile($patron)
             'zip' => $result['postal_code'],
             'city' => $result['city'],
             'country' => $result['country'],
-            'expiration_date' => $this->convertDate($result['expiry_date'] ?? null)
+            'expiration_date' => $this->convertDate($result['expiry_date'] ?? null),
```

Comment:
Isn't convertDate going to return whatever display date format is configured in config.ini, rather than the desired Y-m-d?

Reply:
Oops. Yes. And it's for display purposes anyway. I just stuffed up when updating the wiki page.

Reply:
Wiki fixed.